### PR TITLE
[plugin.audio.itvrs_radio_ice] v0.0.3

### PR DESCRIPTION
**Pull request by remichie**
### Description
ITVRS Radio audio Add-on

v0.0.3  (20-8-2019)
- dropped the usage of script.module.neverwise

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.itvrs_radio_ice/addon.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.itvrs_radio_ice"
+	name="ITVRS - Radio Ice"
+	version="0.0.3"
+	provider-name="Rudi Paret (Remichie)">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.neverwise" version="1.0.8"/>
```
Comment (Rechi) [inline plugin.audio.itvrs_radio_ice/addon.xml]: unused, drop this line

File: plugin.audio.itvrs_radio_ice/default.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+line2 = '  '
+line3 = 'Use Left Menu To Pause Or Stop'
+
+xbmcgui.Dialog().ok(addonname, line1, line2, line3)
+
+class itvrs_radio_ice:
+
+    def __init__(self):
+		
+        xbmc.Player().play('http://stream.zeno.fm/3611cwn8mbruv')
```
Comment (Rechi) [inline plugin.audio.itvrs_radio_ice/default.py]: Calling `xbmc.Player().play(...)` isn't allowed in an plugin.
You have to create a directory listing, which contains one `ListItem`. For that list item you set the path to the above URL.

File: plugin.audio.itvrs_radio_ice/default.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import os
+import xbmcaddon
+import xbmcgui
+
+addon = xbmcaddon.Addon()
+addonname = addon.getAddonInfo('name')
+line1 = 'ITVRS - Radio Ice'
+line2 = '  '
+line3 = 'Use Left Menu To Pause Or Stop'
```
Comment (enen92) [inline plugin.audio.itvrs_radio_ice/default.py]: Would be nice to have such strings localized (i.e in strings.po)

File: plugin.audio.itvrs_radio_ice/default.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+xbmcgui.Dialog().ok(addonname, line1, line2, line3)
+
+class itvrs_radio_ice:
```
Comment (enen92) [inline plugin.audio.itvrs_radio_ice/default.py]: A simple function would be better than instantiating a class object just to play a file (plus you need to use xbmcplugin.setresolvedurl as rechi pointed out)

**Comment by remichie** [comment]
@Rechi, Can you review again. Thanks

**Pull request closed without merging**